docs/model_repository.rst

```diff
@@ -297,6 +297,12 @@ ONNX Models
 ^^^^^^^^^^^
 
 An ONNX model is a single file that by default must be named model.onnx.
+Notice that some ONNX models may not be supported by the inference server
+as they are not supported by the underlying ONNX Runtime (due to either
+using `stale ONNX opset version
+<https://github.com/Microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix>`_
+or containing operators with `unsupported types
```

Comment:
.. or [because they use] operators with `datatypes that are currently unsupported...